scripts/zkai_minimal_d128_block_boundary_wrapper_gate.py

```
@@ -0,0 +1,1075 @@
+                row_id=row["row_id"].replace("_", " "),
+                status=row["status"],
+                scope=row["metric_scope"],
+                proof_json=f"`{row['proof_json_bytes']:,}`" if isinstance(row.get("proof_json_bytes"), int) else "",
+                typed=f"`{row['typed_bytes']:,}`" if isinstance(row.get("typed_bytes"), int) else "",
+                reference=f"`{row['split_reference_bytes']:,}`" if isinstance(row.get("split_reference_bytes"), int) else "",
+                saving=f"`{row['saving_bytes']:,}`" if isinstance(row.get("saving_bytes"), int) else "",
+                ratio=f"`{row['ratio']}`" if row.get("ratio") is not None else "",
+                commitment=row["commitment"],
+                action=row["action"].replace("_", " "),
+            )
+        )
```

Ruff (0.15.13)

[warning] Use `list.extend` to create a transformed list

(PERF401)